docs/FeatureManagement/FeatureFlag.v2.0.0.schema.json

```diff
@@ -0,0 +1,345 @@
+          "percentile": {
+              "$id": "#/properties/allocation/properties/percentile",
+              "type": "array",
+              "title": "Percentile Allocation Collection",
+              "description": "A list of objects, each containing a variant name and percentage range for which that variant should be used.",
+              "items": {
+                  "$id": "#/properties/allocation/properties/percentile/items",
+                  "type": "object",
+                  "title": "Percentile Allocation",
+                  "required": [
+                      "variant",
+                      "from",
+                      "to"
+                  ],
+                  "properties": {
+                      "variant": {
+                          "$id": "#/properties/allocation/properties/percentile/items/properties/variant",
+                          "type": "string",
+                          "title": "Percentile Allocation Variant",
+                          "description": "The name of the variant to use if the calculated percentile for the current user falls in the provided range.",
+                          "pattern": "^(.*)$"
+                      },
+                      "from": {
+                          "$id": "#/properties/allocation/properties/percentile/items/properties/from",
+                          "type": "number",
+                          "title": "Percentile Allocation From",
+                          "description": "The lower end of the percentage range for which this variant will be used."
+                      },
+                      "to": {
+                          "$id": "#/properties/allocation/properties/percentile/items/properties/to",
+                          "type": "number",
+                          "title": "Percentile Allocation To",
+                          "description": "The upper end of the percentage range for which this variant will be used."
+                      }
+                  }
+              }
+          },
+          "seed": {
+              "$id": "#/properties/allocation/properties/seed",
+              "type": "string",
+              "title": "Percentile Allocation Seed",
+              "description": "The value which percentile calculations are based on. Calculated percentiles are consistent across features if the user and seed are consistent.",
+              "default": "",
+              "pattern": "^(.*)$"
+          }
+      }
+    },
+    "telemetry": {
+      "$id": "#/properties/telemetry",
+      "type": "object",
+      "title": "Feature Telemetry Options",
+      "description": "The declaration of options used to configure telemetry for this feature.",
+      "required": [],
+      "properties": {
+        "enabled": {
+          "$id": "#/properties/telemetry/properties/enabled",
+          "type": "boolean",
+          "title": "Telemetry Enabled State",
+          "description": "A flag to enable or disable sending telemetry.",
```

Comment:
The flag itself doesn't enable/disable telemetry. The flag is for telemetry publisher to consume, if installed. How about this?

Indicates if the telemetry of the feature is enabled.

Reply:
I would simplify further to "Indicates if telemetry is enabled."